save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt

```diff
@@ -198,6 +198,16 @@ private fun ChildrenBuilder.displayCpgNodeAdditionalInfo(
 ) {
     div {
         className = ClassName("card card-body p-0")
+        button {
+            className = ClassName("btn p-0 position-absolute")
+            fontAwesomeIcon(faTimesCircle)
+            type = "button".unsafeCast<ButtonType>()
+            onClick = { setSelectedNodeName(null) }
+            style = jso {
+                top = "1%".unsafeCast<Top>()
```

Comment:
why exactly 1%?

Reply:
It was quite hard to come up with an idea of how to place the button in a right position. Iteratively it was found out that 1% is just what is needed. The position does not depend on screen size so it seems to be ok for now

Reply:
Guess even `0%` might be better as far as the vertical position depends on the size of the whole div and it is not that good as far as it can change due to `code` section